versions.json

```diff
@@ -23,6 +23,17 @@
     "version": "3.11.10"
   },
   "3.11-rc": null,
+  "3.12-rc": {
```

Comment:
:thinking: Would also adding a `"3.12": null,` help us get the release automatically, like the `*-rc` keys?

Reply:
I did try, but the current code in `versions.sh` would require some refactoring in order to accomplish that, because it only works for released branches:

https://github.com/docker-library/rabbitmq/blob/dde405ff543c4fa696f146f9b3bc70dcc73bf1f5/versions.sh#L76-L86